### PR TITLE
Issue 68 meetup list

### PR DESCRIPTION
**Pull request by dboydgit**
Add a list of meetups to the home page #68

### CODE REVIEW

**Comment by MarkBennett** [comment]
Hey @dboydgit! Thanks for doing this. I don't know why but I didn't get a notification about your review request.

Looking at it right now!

File: nuxt.config.js [hunk 1]
```diff
@@ -58,7 +58,6 @@ export default {
         /*
          ** You can extend webpack config here
          */
-        extend(config, ctx) {},
```
Comment (MarkBennett) [inline nuxt.config.js:61]: What was the rational for removing this? I might not understand Vue well enough but I couldn't see how it relates to the other changes here.

**Review by MarkBennett — COMMENTED** [review]
Looks great @dboydgit! Thanks for all your hard work on this tracking down all the images and meetup details.

I did have one question about a config change, and was also wondering if you know why when you click on a meetup and then go back in your history it jumps to the top of the page rather than going back to where you'd expect it to in the meetup section? Is this because of the way we're including the links?

File: nuxt.config.js (hunk 1 above)
Comment (dboydgit) [reply nuxt.config.js:61]: Oh it just want being used or doing anything. I think it was there as a default when you set the project up.

**Comment by dboydgit** [comment]
> Looks great @dboydgit! Thanks for all your hard work on this tracking down all the images and meetup details.
> 
> I did have one question about a config change, and was also wondering if you know why when you click on a meetup and then go back in your history it jumps to the top of the page rather than going back to where you'd expect it to in the meetup section? Is this because of the way we're including the links?

I never noticed that link behaviour. I think it's related to nuxt I saw a few hits when I googled it, I would suggest we open another issue to look into it. If you look at the links on the board page they have the same issue. It's likely happening on any links with a dynamic `:href` attr.

File: nuxt.config.js (hunk 1 above)
Comment (MarkBennett) [reply nuxt.config.js:61]: Ok. Sounds good!

**Review by MarkBennett — APPROVED** [review]
Thanks for the work on this @dboydgit! Ship it! 🚢

**Pull request merged**